test/e2e/helpers/utils.js

```diff
@@ -0,0 +1,28 @@
+export function getHugeMySQL(minCharts, columnName) {
```

Comment:
seems like this is needed for a single case, move to the file where it is used

Reply:
```
function generateLongString(length = 1000){
   return new Array(length).fill('a').join('')
}
```

Reply:
\+